### ISSUE TITLE
Firefox: Unable to connect to host 127.0.0.1 on port 7056 after 45000 ms

### ISSUE THREAD

**Issue by elgalu** [SeleniumHQ/selenium#2442]
## Meta -
OS Host: Ubuntu 14.04.3 LTS 3.19.0-28-generic (Travis worker)
OS Container: Ubuntu 16.10 yakkety 20160708 (Docker container)
Selenium Version: standalone 2.53.1
Browser:  Firefox
Browser Version:  47.0.1

## Expected Behavior -
Can run N Firefox tests in parallel on the same network interface.

## Actual Behavior -
Some sort of race condition on ports 7054 ... 7056 giving errors:

- [Message: Unable to connect to host 127.0.0.1 on port 7056 after 45000 ms][build233]
- `selenium.common.exceptions.WebDriverException: Message: Unable to bind to locking port 7054 within 45000 ms`
- [org.openqa.selenium.internal.SocketLock.lock][build234]

By race condition I mean it happens sometimes but most time works fine.

Seems somehow related to [issue-archive/7272](https://github.com/seleniumhq/selenium-google-code-issue-archive/issues/7272) and/or [issue-archive/4790](https://github.com/seleniumhq/selenium-google-code-issue-archive/issues/4790)

## Steps to reproduce -
See
- [failed travis build 233][build233]
- [failed travis build 234][build234]

Or: install Docker and Docker-Compose and:

```sh
git clone git@github.com:elgalu/docker-selenium.git
cd docker-selenium
./test/script_scenario_compose_N_N
```

**UPDATE 1** It only seems to happen on Ubuntu [yakkety-20160708](https://hub.docker.com/r/library/ubuntu/) or xenial-20160706 however I rolled back to xenial-20160629 and I cannot reproduce the issue anymore.

**UPDATE 2** False alarm regarding Ubuntu versions, it [also happened again on xenial-20160629](https://travis-ci.org/elgalu/docker-selenium/jobs/143606031#L5750)

[build233]: https://travis-ci.org/elgalu/docker-selenium/jobs/143561228#L5727
[build234]: https://travis-ci.org/elgalu/docker-selenium/jobs/143563522#L5417

**Comment by lukeis** [comment]
i'm going to mark this as a duplicate of #1915

there's something going on and it doesn't appear to be OS specific... but we're also not likely to spend much time on it since FirefoxDriver will be non-functioning with FF48+ and the path forward is to use geckodriver

**Comment by lukeis** [comment]
related PR #2410
